package.json

```diff
@@ -5,13 +5,13 @@
   "license": "Apache-2.0",
   "author": "Google Inc",
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=8.13.0"
   },
   "repository": "googleapis/nodejs-dataproc",
-  "main": "src/index.js",
+  "main": "build/src/index.js",
   "files": [
-    "protos",
-    "src",
+    "build/protos",
+    "build/src",
     "AUTHORS",
```

Comment:
Could I trouble you to delete `AUTHORS` and `COPYING` here?  They aren't needed.

Reply:
Done